src/modules/statsd/statsd.c

```diff
@@ -278,42 +279,51 @@ static int ki_statsd_stop_with_labels(sip_msg_t *msg, str *key, str *labels)
 
 static int func_incr(struct sip_msg *msg, char *key)
 {
-	return statsd_count(key, "+1", NULL);
+	return convert_result(statsd_count(key, "+1", NULL));
 }
 
 static int func_incr_with_labels(struct sip_msg *msg, char *key, char *labels)
 {
-	return statsd_count(key, "+1", labels);
+	return convert_result(statsd_count(key, "+1", labels));
 }
 
 static int ki_statsd_incr(sip_msg_t *msg, str *key)
 {
-	return statsd_count(key->s, "+1", NULL);
+	return convert_result(statsd_count(key->s, "+1", NULL));
 }
 
 static int ki_statsd_incr_with_labels(sip_msg_t *msg, str *key, str *labels)
 {
-	return statsd_count(key->s, "+1", labels->s);
+	return convert_result(statsd_count(key->s, "+1", labels->s));
 }
 
 static int func_decr(struct sip_msg *msg, char *key)
 {
-	return statsd_count(key, "-1", NULL);
+	return convert_result(statsd_count(key, "-1", NULL));
 }
 
 static int func_decr_with_labels(struct sip_msg *msg, char *key, char *labels)
 {
-	return statsd_count(key, "-1", labels);
+	return convert_result(statsd_count(key, "-1", labels));
 }
 
 static int ki_statsd_decr(sip_msg_t *msg, str *key)
 {
-	return statsd_count(key->s, "-1", NULL);
+	return convert_result(statsd_count(key->s, "-1", NULL));
 }
 
 static int ki_statsd_decr_with_labels(sip_msg_t *msg, str *key, str *labels)
 {
-	return statsd_count(key->s, "-1", labels->s);
+	return convert_result(statsd_count(key->s, "-1", labels->s));
+}
+
+static int convert_result(bool result)
+{
+	if(result == false) {
+		return -1;
+	}
+
+	return 0;
```

Comment:
Shouldn't the above line be `return 1`? Or I misunderstood the purpose.

Reply:
Yes, correct, we noticed this in our local build but haven't updated here yet, will fix

Reply:
Updated https://github.com/kamailio/kamailio/compare/3bbed6d62a2ef3a3fbcbf4d8ba125ab95aec28dd..ee772af9e7acbdac14247010e1e7ecf539d759ba